health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/AppConfigurationWrapper.js

```
@@ -116,12 +116,20 @@ const reducer = (state = initialState, action, updateLocalization) => {
               ...item,
               cards: item?.cards?.map((j, k, c) => {
                 if (j.header === action.payload.currentCard?.header) {
+                  const regex = new RegExp(`^${item?.name}_${j?.header}_NEW_FIELD_(\\d+)$`);
+                  const maxCounter = j.fields
+                    .map((f) => {
+                      const match = f.jsonPath && f.jsonPath.match(regex);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)